README.md

```diff
@@ -74,6 +74,7 @@ AutoGluon also provides state-of-the-art tools for neural hyperparameter and arc
 
 - [General introduction into HNAS](https://www.youtube.com/watch?v=pB1LmZWK_N8&feature=youtu.be)
 - [Introduction into HNAS with AutoGluon](https://www.youtube.com/watch?v=GJVwUyVWZas)
+- [Examples](https://github.com/awslabs/autogluon/tree/master/examples/nas/hnas)
```

Comment:
if you're trying to link to the directory of the file you created here, this should be `https://github.com/awslabs/autogluon/tree/master/examples/hnas` right?

Reply:
good catch! thanks